.gitignore

```diff
@@ -4,3 +4,4 @@ package-lock.json
 yarn.lock
 coverage
 analysis.json
+.idea
```

Comment:
I don't think we have this in any other repos. Maybe it would be a good idea to add this (and think is there something else that should be added also) but it should be done consistently in all component repos. I'd leave it out of this PR before we discuss this more with the team.

Reply:
I recently switched to using IntelliJ Idea for both Java and frontend development, and it adds this folder for IDE metadata. The reason why we don't have this in our .gitignores is that no one in the team has previously worked on frontend stuff with IntelliJ.

It sounds definitely a good idea to have it in all the repos, since IntelliJ is a common tool for web development (although not in our team). Although it's more common for Java development, the ultimate edition contains basically all the same features as JetBrains WebStorm.

But making a separate PR for all repos and then cherry-picking it to different branches is not worth it IMO. I'd prefer just adding it with on-demand basis #ffs

Reply:
Removed the line.
As concluded in Slack, I will use global gitgnore for this purpose.